### ISSUE TITLE
Adguard v0.107.0-b.13 snap DHCP server blocked by apparmor

### ISSUE THREAD

**Issue by clhuby** [AdguardTeam/AdGuardHome#3736]
### Issue Details

* **Version of AdGuard Home server:**
  * v0.107.0-b.13 snap
* **How did you install AdGuard Home:**
  * Snapcraft (Ubuntu appliance)
* **If it's a router or IoT, please write device model:**
  * Raspberry Pi 3 Model B
* **Operating system and version:**
5.3.0-1030-raspi2 (buildd@bos02-arm64-008) (gcc version 7.5.0 (Ubuntu/Linaro 7.5.0-3ubuntu1~18.04)) #32~18.04.2-Ubuntu SMP

### Expected Behavior
DHCP server enables and provides DHCP service to network.

### Actual Behaviour
Error permission denied when click enable, then appears enabled but does not serve IPs.

### Log
Oct 15 07:25:29 localhost adguard-home.adguard-home[746]: 2021/10/15 07:25:29.810412 [info] can't check for static ip; assuming machine has static ip and going on
Oct 15 07:25:29 localhost audit[746]: AVC apparmor="DENIED" operation="create" profile="snap.adguard-home.adguard-home" pid=746 comm="AdGuardHome" family="packet" sock_type="raw" protocol=0 requested_mask="create" denied_mask="create"
Oct 15 07:25:29 localhost adguard-home.adguard-home[746]: 2021/10/15 07:25:29.811373 [info] DHCP: POST /control/dhcp/set_config: enabling dhcp: starting dhcp server: dhcpv4: creating raw udp connection: permission denied
Oct 15 07:25:29 localhost kernel: audit: type=1400 audit(1634282729.809:24): apparmor="DENIED" operation="create" profile="snap.adguard-home.adguard-home" pid=746 comm="AdGuardHome" family="packet" sock_type="raw" protocol=0 requested_mask="create" denied_mask="create"

</details>

### Additional Information
This occured after the last snap refresh, I have also tried a fresh install of the appliance with the dhcp client disabled and get the same issue.

**Comment by bntsp** [comment]
Same problem here.

**Comment by uldirien** [comment]
same here as well

**Comment by ainar-g** [comment]
@uldirien, see #3976.  We'll post an update about the situation there soon.

**Comment by Sil3ntVip3r** [comment]
This now appears closable as a resolved duplicate of #3976.

The reported denial is the same raw-packet socket permission problem.  In
#3976, adding the `network-control` interface fixed the confirmed cases, and
the Snap Store subsequently granted the required automatic permission.
Current `master` still declares `network-control` specifically for DHCP raw
sockets, and the current stable Snap is being published for all supported
architectures.

If this exact AppArmor denial recurs on a current Snap, the useful new evidence
would be `snap connections adguard-home` plus startup and DHCP logs.  The
original v0.107.0-beta incident itself was resolved through the Snap Store
permission change.
